irm/api/Snippets/CreateIncident.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) 2019 Google LLC.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// [START monitoring_irm_create_incident]
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.Irm.V1Alpha2;
+using Google.Protobuf.WellKnownTypes;
+using static Google.Cloud.Irm.V1Alpha2.Incident.Types;
+using System;
+
+class IrmCreateIncident
+{
+    public Signal CreateIncidentWithSignal(string projectId = "YOUR-PROJECT-ID")
+    {
+        // Create client
+        IncidentServiceClient incidentServiceClient =
+            IncidentServiceClient.Create();
+
+        // Manually create a signal.

```

Comment:
We should also explain what signal means in this context

Reply:
The code will be embedded in a doc which explains what signal means.